external/PATCH-01-STRATUM

```diff
@@ -2901,13 +2905,19 @@ index 9f4bb3a9..e0154d94 100644
 -    RETURN_IF_BFRT_ERROR(table->tableUsageGet(
 -        *bfrt_session, bf_dev_target,
 -        bfrt::BfRtTable::BfRtTableGetFlag::GET_FROM_SW, &entries));
-+    tdi_status = table->usageGet(
++    size_t table_size;
++    tdi_status = table->sizeGet(
 +        *tdirt_session, tdi_dev_target,
-+        *flags, &entries);
++        *flags, &table_size);
++    entries = table_size;
    }
  
    table_keys->resize(0);
-@@ -507,14 +540,21 @@ template <typename T>
+   table_datums->resize(0);
+-  if (entries == 0) return ::util::OkStatus();
++ if (entries == 0) {
```

Comment:
We have two checks for same condition here. Remove the second one

Reply:
I guess the line 2917 is modified to new one, anyways I will revert it to original one